src/test.ts

```diff
@@ -0,0 +1,176 @@
+import os from 'os';
```

Comment:
Maybe we could rename `test.ts` to `sanitize-arguments.ts` this might be nearer to what we want and avoid any confusion (for example with automated tests)?

Reply:
yes that is a better name I will change it

Reply:
should I rename the functions as well to like sanitize-mwUrl etc.

Reply:
Yes, but ok but please keep using `_` as separator